### PR TITLE
`Stringable` interface does not implement `Stringable`

### PR DESCRIPTION
**Pull request by herndlm**
that would be another circular reference again :)

https://3v4l.org/QqBDN

### CODE REVIEW

File: src/Reflection/ReflectionClass.php [hunk 1]
```diff
@@ -1218,7 +1218,7 @@ private function addStringableInterface(array $interfaces): array
         /** @psalm-var class-string $stringableClassName */
         $stringableClassName = Stringable::class;
 
-        if (array_key_exists($stringableClassName, $interfaces)) {
+        if (array_key_exists($stringableClassName, $interfaces) || ($this->isInterface && $this->getName() === $stringableClassName)) {
```
Comment (herndlm) [inline src/Reflection/ReflectionClass.php:1221]: do I need to check if this is an interface or can this be simplified?

**Comment by herndlm** [comment]
Weird, the escaped mutant should not be caused by my changes 🤔

**Comment by Ocramius** [comment]
It could be that the added conditionals make previous conditionals redundant here :)

**Comment by kukulich** [comment]
Shit. I've merge it to wrong branch.

**Comment by kukulich** [comment]
@Ocramius I'm not sure what's the best way how to solve the merge to wrong branch 😞 Should I open duplicate PR to right branch? Or just cherry-pick the commits?

**Comment by ondrejmirtes** [comment]
I wouldn't worry about it too much - the next release can be 6.1.0 and no one can oppose that.

**Comment by herndlm** [comment]
sorry, I didn't notice either that 6.1.x was the target. I think it was the default branch and I didn't check :/

**Comment by Ocramius** [comment]
No big issue: we'll release `6.1.0` then.

**Pull request merged**